t/app/controller/report_inspect.t

```diff
@@ -247,6 +249,18 @@ FixMyStreet::override_config {
     ALLOWED_COBRANDS => [ 'oxfordshire', 'fixmystreet' ],
     BASE_URL => 'http://fixmystreet.site',
 }, sub {
+    subtest "test resending when send_method changes" => sub {
+        $mech->get_ok("/report/$report3_id");
+        # Then change the category to the other category within the same council,
+        # which should cause it to be resent because it has a different send method
+        $mech->submit_form(button => 'save', with_fields => { category => 'Sheep', include_update => undef, });
+
+        $report3->discard_changes;
+        is $report3->category, "Sheep", "Report in correct category";
+        is $report3->whensent, undef, "Report marked as unsent";
+        is $report3->bodies_str, $oxon->id, "Reported to OCC";
+    };
+
```

Comment:
Unless it's elsewhere and I'm missing it it'd be nice to see a test that shows changing to a category with the same send method doesn't resend.